src/Frontend/Assets.php

```diff
@@ -62,12 +69,18 @@ class Assets
      */
     protected $fileSourceOverrides = [];
 
-    public function __construct(string $name, Filesystem $assetsDir, string $cacheDir = null, array $lessImportDirs = null)
+    /**
+     * @var array
+     */
+    protected $customFunctions = [];
+
+    public function __construct(string $name, Filesystem $assetsDir, string $cacheDir = null, array $lessImportDirs = null, array $customFunctions = null)
```

Comment:
Why not have `[]` the default for `customFunctions`?

Reply:
```suggestion
    public function __construct(string $name, Filesystem $assetsDir, string $cacheDir = null, array $lessImportDirs = null, array $customFunctions = [])
```